From f3f47edae9fec1ca9a9d8e3c07da6a6b6ffc5a72 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Mon, 26 Apr 2021 01:31:27 +0200
Subject: arm64: dts: sun50i-a64-pinephone: Don't make lradc keys a wakeup
 source

This allow the crust to reach lower suspend state.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index e39b070deec9..6f534ff774fb 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -533,7 +533,7 @@ &i2s2 {
 
 &lradc {
 	vref-supply = <&reg_aldo3>;
-	wakeup-source;
+//	wakeup-source;
 	status = "okay";
 
 	button-200 {
-- 
2.35.3

